cuda_pathfinder/cuda/pathfinder/_version.py

```diff
@@ -1,4 +1,4 @@
 # SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
Are we committed to keeping pathfinder in sync between `main` and `12.9.x`? It's a bit funky that we'll have two different sources of cuda-pathfinder 1.1.0 that aren't the same.

Reply:
Great question. I think you've discovered a flaw in the current CI design... we weren't prepared to always fetch the pathfinder wheel from the main branch. So right now with the backport branch we do have two different sources... 

As a bandaid, I can manually backport all changes in the pathfinder from the main branch. Later we can improve the CI design to fetch pathfinder from the main branch in the `12.9.x` CI (just like how we fetch 12.9.x wheels from the backport branch in the `main` CI, but in the reversed direction). WDYT?

Reply:
> As a bandaid, I can manually backport all changes in the pathfinder from the main branch. 

This is done in commit [8629941](https://github.com/NVIDIA/cuda-python/pull/806/commits/8629941c8dfa8674f82056f5a5c6a980fefaa54f)

Reply:
That sounds good to me.